content/ja/handbook/customer-success/csm/segment/csa/index.md

```
@@ -0,0 +1,126 @@
+translated_at: "2026-04-26T00:00:00Z"
+translator: claude
+stale: false
+---
+
+[CSM ハンドブックホームページ](/handbook/customer-success/csm/)で、CSM に関連するその他のハンドブックページをご覧ください。
+
+GitLab では、顧客の成功が私たちの成功であると理解しています。そのため、カスタマーサクセスアーキテクト（CSA）ロールを導入しました。CSA は、カスタマーサクセス内で最高評価の技術プロダクトエキスパートとして、GitLab との旅全体を通じてトップレベルの専門知識とガイダンスを提供します。
+
+## CSA エンゲージメントの仕組み
+
+CSA エンゲージメントは、GitLab のサブスクリプションと共に購入する有料サービスオファリングである GitLab の **Advanced** および **Signature** [サクセスティア](/handbook/customer-success/success-services/)を通じて利用できます。CSA は、あなたのアカウントに割り当てられた専任の指名リソースとして、継続的な技術ガイダンスを提供し、戦略的ゴールに合わせた構造化された[アクセラレータプログラム](#accelerator-lifecycle)をリードします。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)